test/osutils.jl

```diff
@@ -28,6 +28,9 @@
     else
         @test Sys.windows_version() >= v"1.0.0-"
     end
+
+    @test !Sys.detectwsl(:Windows)
+    @test !Sys.detectwsl(:Linux)
```

Comment:
Won't this test fail if you run it under WSL?

Reply:
I don't know how we are going to test it. Yes these tests will fail inside WSL how are we going to test it?

Reply:
I don't know how we are going to test it. Yes these tests will fail inside WSL how are we going to test it?

Reply:
I don't know how we are going to test it. Yes these tests will fail inside WSL how are we going to test it?

Reply:
I would test that it's not true when the argument is not `:Linux` since that's always correct and then add this test for the zero-argument version:
```jl
if !Sys.islinux()
    @test !Sys.detectwsl()
end
```

Reply:
Should the Sys.detectwsl(:Windows) test be removed?